cnab/app/kab/manifest.yaml

```diff
@@ -106,6 +106,22 @@ spec:
       selector:
         matchLabels:
           app: webhook
+  - name: riff-system
+    namespace: riff-system
+    path: https://storage.googleapis.com/projectriff/riff-system/riff-system-0.1.0-snapshot.yaml
```

Comment:
riff system is still evolving quite a bit at this stage, and this resource will mutate with those changes. If we need any sort of compatibility guarantee we should use a build from a specific gitsha instead. But of course then the question turns to how to maintain that revision.